server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailSetMethodContract.scala

```diff
@@ -7507,6 +7507,142 @@ trait EmailSetMethodContract {
 
   }
 
+  @Disabled("Bug of james-mime4j when pasing mimeMessages https://github.com/linagora/james-project/issues/5086")
```

Comment:
Where the problem is? In Email/get?

Reply:
Can't we handle the exception in Scala code for the time being?

Reply:
yep, Email/get
it returns incorrect `to` property response

Reply:
> Can't we handle the exception in Scala code for the time being?

We can handle it by filtering addresses with not value ">" 
Is it acceptable?

![image](https://github.com/apache/james-project/assets/81145350/518dec8c-9fa1-4613-8f24-75deb29158c3)

Reply:
As expressed in https://github.com/linagora/james-project/issues/5086#issuecomment-1961084516 this corner on't happen with Twake mail in practice so it will not be a blocker...